assets/app/styles/_core.less

```diff
@@ -104,6 +104,16 @@ html, body {
       margin-bottom: 0;
     }
   }
+  .label-filter {
```

Comment:
`label-filter` we probably should.

`active-filters` is styled in kubernetes-ui so that it looks correct when it appears to the right of the filter text field. It's showing below in our UI, so I made the change. Honestly, I'm not sure I like the way the CSS is done for label-selector in kubernetes-ui. If I'm reading it right, it seems to be making assumptions about when the active filter wraps below the text field that are probably incorrect in other places.